src/App.js

```diff
@@ -54,7 +57,7 @@ function App() {
           handleSubmit={handleSubmit}
           formValid={formValid}
         />
-        <LoadingIndicator />
+        <LoadingIndicator status={loading} />
```

Comment:
instead of making a new component you can simply conditionaly render the spinner for example:
```suggestion
        loading && <Loader type="ThreeDots" color="#eebb22" height={80} width={80} />
```

Reply:
Thanks. Implemented in latest commit!